### PR TITLE
[SMALLFIX] Port MasterClientIntegrationTest

### PR DESCRIPTION
**Pull request by uronce-cc**
The ported test works, but one test case is comment out because no counterpart API for `MasterClient#user_getWorker` can be found in new master client.

### CODE REVIEW

File: integration-tests/src/test/java/tachyon/master/next/MasterClientIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+ * Though its name indicates that it provides the tests for {@link tachyon.master.MasterClient},
+ * this class is more like unit-testing the internal implementation of tachyon Master via a
+ * {@link tachyon.master.MasterClient}, and thus it depends on many components in tachyon.master.
+ * As a result, we place MasterClient in tachyon-common and this test in tachyon-integration-tests.
+ *
+ * <p>
+ * TODO: Rename this class.
+ *
+ */
+public class MasterClientIntegrationTest {
```
Comment (gpang) [inline integration-tests/src/test/java/tachyon/master/next/MasterClientIntegrationTest.java]: This should be FileSystemMasterClientIntegrationTest (and there would probably be another BlockMasterClientIntegrationTest).

**Comment by gpang** [comment]
I just had a minor comment. Thanks!

**Comment by uronce-cc** [comment]
@gpang PTAL

**Comment by gpang** [comment]
LGTM

**Pull request merged**